apps/fluent-tester/src/testPages.ts

```diff
@@ -93,6 +94,12 @@ export const tests: TestDescription[] = [
     testPage: HOMEPAGE_CONTEXTUALMENU_BUTTON,
     platforms: ['android', 'macos', 'win32'],
   },
+  {
+    name: 'Corner Radius Tokens',
+    component: CornerRadiusTokensTest,
+    testPage: HOMEPAGE_CORNERRADIUS_BUTTON,
+    platforms: ['android', 'ios', 'macos', 'win32'],
```

Comment:
Might as well add 'windows' too?

Reply:
Sure. Do we also want 'web'? I saw that some components have 'web' as well, not really sure what that means